InstanceManager.js

```diff
@@ -268,37 +269,52 @@ class InstanceManager {
       compactionKeep = process.env.AGENCY_COMPACTION_KEEP;
     }
     for (var i = 0; i < size; i++) {
```

Comment:
Would it make sense to start the instances in parallel instead of one after the other?